build.py

```diff
@@ -294,6 +295,41 @@ def build_project(root):
     execute(['python', 'scripts/main.py'])
 
 
+def setup_appintro(root):
+    """ Check if appIntro is turned on for the region, and, if so
+    setup the project so that appIntro page becomes the root, and
+    the region page can be embedded.
+    """
+    region_json_path = os.path.join(root, FRAMEWORK_REPO, 'src', 'region.json')
+
+    with open(region_json_path) as r:
+        region_json = json.load(r)
+
+        if region_json['appIntro']:
```

Comment:
I don't think this will work if `appIntro` is not included in `region.json`, which it won't for all existing regions.  It's also marked as optional in the regionSchema.  Consider `region_json.get('appIntro', False)` or similar.

Reply:
Good call. Updated.